src/ergw_sup.erl

```diff
@@ -30,6 +30,7 @@ start_link() ->
 %% ===================================================================
 
 init([]) ->
+    ok = ergw_node_selection_cache:init(),
```

Comment:
Not sure but maybe make sense create `ergw_node_selection_cache` as worker

Reply:
That was the initial implementation indeed. However I wanted to benefit from the ets concurrency for lookup requests without serialization, it ended up being an empty gen_server process just holding the ETS table without any added value.